db/migrate/20200504201055_add_mail_settings_to_users.rb

```diff
@@ -0,0 +1,13 @@
+class AddMailSettingsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users,
+               :receives_results_emails,
+               :boolean,
+               default: true
```

Comment:
Add `null: false` to both of these columns. Also, I think for now it's best to set the default to `false` rather than `true`. (This will make more sense for TAs/instructors, but also since this feature is new we'll try it out as opt-in rather than opt-out.)